app/views/revaluation.scala.html

```diff
@@ -44,40 +44,40 @@ <h1>@Messages("gmp.revaluation.question")</h1>
             @views.html.helpers.groupError(revaluationForm("revaluationDate"))
               @if(revaluationForm.hasGlobalErrors) {
                 @revaluationForm.globalErrors.zipWithIndex.map { case (error, index) =>
-                    <span style="display: block" class="error-notification" id="global-error-@index">@error.message</span>
+                    <span style="display: block" class="govuk-error-message" id="global-error-@index">@error.message</span>
                 }
               }
 
             <p class="form-hint">@Messages("gmp.date.example")</p>
 
             <fieldset id="revaluationDate">
 
-                <legend id="reval-date" class="visuallyhidden">
+                <legend id="reval-date" class="govuk-visually-hidden">
                     @Messages("gmp.revaluation.question")
                 </legend>
 
                 <div class="form-group">
                     @gmpInput(revaluationForm("revaluationDate.day"),
                     '_label -> Html(Messages("gmp.day")),
-                    '_id -> "revaluationDate_day",
+                    '_id -> "revaluationDate-day",
                     '_maxLength -> 2)
                 </div>
                 <div class="form-group">
                     @gmpInput(revaluationForm("revaluationDate.month"),
                     '_label -> Html(Messages("gmp.month")),
-                    '_id -> "revaluationDate_month",
+                    '_id -> "revaluationDate-month",
                     '_maxLength -> 2)
                 </div>
                 <div class="form-group">
                     @gmpInput(revaluationForm("revaluationDate.year"),
                     '_label -> Html(Messages("gmp.year")),
-                    '_id -> "revaluationDate_year",
+                    '_id -> "revaluationDate-year",
                     '_maxLength ->4,
                     '_inputClass -> "form-control year-input")
                 </div>
-                <br/>
-                <br/>
-                <br/>
+                <hr class="govuk-section-break" aria-hidden = true>
+                <hr class="govuk-section-break" aria-hidden = true>
+                <hr class="govuk-section-break" aria-hidden = true>
 
             </fieldset>
 
```

Comment:
This page is okay if it is not a page within this journey.  If it is, then gmp_main_old should no longer be used

Reply:
I have removed gmp_main_old for the pages which were on my journey, others will be removed later who so ever picks the tickets